src/modules/storageMethods.js

```diff
@@ -0,0 +1,82 @@
+import Task from './taskConstructor.js';
+
+export default class Methods {
+  static setIndex() {
+    const storage = localStorage.getItem('tasks');
+    const tasks = storage ? JSON.parse(storage) : [];
+    if (tasks !== []) {
+      tasks.forEach((element, index) => { element.id = index; });
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+    return tasks;
+  }
+
+  static add(description) {
+    if (!description) return -1;
+    const storage = localStorage.getItem('tasks');
+    const tasks = storage ? JSON.parse(storage) : [];
+    const task = new Task(description, tasks.length);
```

Comment:
- [ ] Kindly notice that if a 5th task is added to the list, the index of that task should equal to 5, so please fix this by adding one to the task.length.

```javascript
const task = new Task(description, tasks.length + 1);
```